cdap-hbase-compat-0.94/src/main/java/co/cask/cdap/data2/transaction/queue/coprocessor/hbase94/HBaseQueueRegionObserver.java

```diff
@@ -79,7 +80,8 @@ public void start(CoprocessorEnvironment env) {
         this.prefixBytes = HBaseQueueAdmin.SALT_BYTES;
       }
 
-      namespaceId = HBaseQueueAdmin.getNamespaceId(tableName);
+      namespaceId = HBase94TableUtil.fromTableName(tableDesc.getNameAsString()).getNamespace().getId();
```

Comment:
RegionObserver shouldn't have dependency on table util. Move that method to `HTableNameConverter`, which is what it is intended for.